core_backend/app/schemas.py

```diff
@@ -14,6 +16,17 @@ class UserQueryBase(BaseModel):
     query_text: str
     query_metadata: dict = {}
 
+    model_config = ConfigDict(from_attributes=True)
```

Comment:
What does this do? Could not understand the [docs](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.from_attributes) :/

Reply:
It allows you to access the properties with either dot notation or like a dict.